services/web/client/source/class/qxapp/component/widget/NodesTree.js

```diff
@@ -132,16 +133,28 @@ qx.Class.define("qxapp.component.widget.NodesTree", {
       return toolbar;
     },
 
+    __getOneSelectedRow: function() {
+      const selection = this.__tree.getSelection();
+      if (selection && selection.toArray().length > 0) {
+        return selection.toArray()[0];
+      }
+      return null;
+    },
+
     __buildTree: function() {
       let tree = new qx.ui.tree.VirtualTree(null, "label", "children").set({
         openMode: "none"
       });
       tree.addListener("dbltap", e => {
-        let selection = this.__tree.getSelection();
-        let currentSelection = selection.toArray();
-        if (currentSelection.length > 0) {
-          let selectedRow = currentSelection[0];
-          this.fireDataEvent("nodeDoubleClicked", selectedRow.getNodeId());
+        const currentSelection = this.__getOneSelectedRow();
+        if (currentSelection) {
+          this.fireDataEvent("nodeDoubleClicked", currentSelection.getNodeId());
+        }
+      }, this);
+      tree.addListener("tap", e => {
+        const currentSelection = this.__getOneSelectedRow();
+        if (currentSelection) {
+          this.fireDataEvent("changeSelectedNode", currentSelection.getNodeId());
         }
```

Comment:
it seems odd that you have to actually go to this level ... doesn't the selection model issue events when the selection is changed ?

Reply:
That's the 'workaround' for selecting the item without having to click/tap on the empty spaces of the ListItem

Reply:
ah ok